README.md

```diff
@@ -1,13 +1,20 @@
-# <img src='https://raw.githack.com/FortAwesome/Font-Awesome/master/svgs/solid/calendar.svg' card_color='#22a7f0' width='50' height='50' style='vertical-align:bottom'/> Date and Time
+# Date and Time
 Get the time, date, day of the week
 
 ## About 
 Get the local time or time for major cities around the world.  Times
 are given in 12-hour (2:30 pm) or 24-hour format (14:30) based on the
-Time Format setting at [Home](https://home.mycroft.ai/#/setting/basic)
+Time Format setting at your [mycroft.conf](https://github.com/OpenVoiceOS/ovos-config/blob/dev/ovos_config/mycroft.conf#L33):
```

Comment:
I think this should point to the local `~/ovos/.config/mycroft.conf` file and can point to the [docs here](https://openvoiceos.github.io/community-docs/config/) for more reference.

Reply:
the rest can be left with a comment to `add this to config`